src/main/java/gov/usgs/wma/waterdata/timeseries/TimeSeriesDao.java

```diff
@@ -26,8 +26,8 @@ public String getTimeSeries(String featureId, String timeSeriesId) {
 
 	public String getTimeSeries(String collectionId, String featureId, String timeSeriesId) {
 		Map<String,String> params = new HashMap<>();
-		params.put(COLLECTION_KEY, collectionId);
-		params.put(FEATURE_KEY, featureId);
+		params.put(CollectionsDao.COLLECTION_KEY, collectionId);
```

Comment:
Code is inconsistent on the include of these constants. Sometimes the import is of the constant, other times just the containing class

Reply:
That is a pattern not an inconsistency. I am open to moving them. The reason they are in the places they are is because high level URLs need to use and define there own constants and then the lower level time series constant is only necessary at that level. It is sensible. However, if you want a constants class to hold them then we could do that.